libraries/resource_providers/api500/synergy/datacenter_provider.rb

```diff
@@ -0,0 +1,22 @@
+# (c) Copyright 2017 Hewlett Packard Enterprise Development LP
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed
+# under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the License for the
+# specific language governing permissions and limitations under the License.
+
+require_relative '../../api300/synergy/datacenter_provider'
+
+module OneviewCookbook
+  module API500
+    module Synergy
+      # Datacenter API500 Synergy resource provider methods
```

Comment:
Why does the Synergy has a comment so different from the C7000?
`Datacenter API500 Synergy resource provider methods`
vs
`Datacenter API500 C7000 provider`
Suggestion: standardize it